docs/user_guide/method/method.rst

```diff
@@ -0,0 +1,79 @@
+.. _method_documentation:
```

Comment:
This file will be updated to reflect proper method documentation in another pull request